Automattic/wp-calypso
# Subscriptions management: Enhance subscriptions tab switcher mobile UX

Resolves https://github.com/Automattic/wp-calypso/issues/76576


<p align="center">
  <table>
    <tr>
      <td>
        <img src="https://github.com/Automattic/wp-calypso/assets/2019970/694ce642-21b7-43bd-9d9a-94f0f0ce35b7" alt="After" width="400" />
        <p align="center"><em>Before (scrolled to the right)</em></p>
      </td>
    </tr>
    <tr>
      <td>
        <img src="https://github.com/Automattic/wp-calypso/assets/2019970/be879304-c715-439b-81bd-97b26f091ad0" 
alt="After" width="400" />
        <p align="center"><em>After</em></p>
      </td>
      <td>
        <img src="https://github.com/Automattic/wp-calypso/assets/2019970/32120054-153e-4154-8682-c7bcb77e17bb" alt="After (with dropdown open)" width="400" />
        <p align="center"><em>After (with dropdown open)</em></p>
      </td>
    </tr>
  </table>
</p>

## Proposed Changes

* Revert to using the native mobile behavior of the `SectionNavItem` component, i.e. to use a select-dropdown for tab selection in mobile view

## Testing Instructions



1. Check out the PR.
2. Navigate to `client/server/pages/index.js` and temporarily add `return next();` to the line `833`:

![Markup on 2023-04-28 at 17:50:44](https://user-images.githubusercontent.com/25105483/235195033-4902bb9d-db57-43a4-b295-2d90feb6731f.png)

3. Build the app.
4. Log in to a test WordPress.com account that has some Sites subscriptions.
5. Navigate to http://calypso.localhost:3000/subscriptions/sites with a web view of horizontal resolution lower than `480px`.
6. You should now see the select-dropdown instead of tabs row. Ensure that scaling back to a resolution with width greater than 480px displays the tabs row in full. 

_Credits to @ivan-ottinger for the above testing instructions 🙇_ 

## Pre-merge Checklist
